src/Aml/Service.php

```diff
@@ -47,7 +48,7 @@ public function __construct(string $sdkId, PemFile $pemFile, Config $config)
      *
      * @return \Yoti\Aml\Result
      *
-     * @throws \Yoti\Exception\AmlException
+     * @throws YotiException
```

Comment:
Not sure I understand this change, we're doing `throw new AmlException` below, so does that now mean this should still be annotated with `@throws AmleException`?

Reply:
Fixed.